sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/vcpkg/vcpkg.json

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+{
+  "name": "azure-messaging-eventhubs-checkpointstore-blob-cpp",
+  "version-semver": "@AZ_LIBRARY_VERSION@",
+  "description": [
+    "Microsoft Azure Messaging Event Hubs SDK for C++ Blob Checkpoint Store",
+    "This library provides an Azure-Storage-Blobs based implementation of an Azure Messaging Event Hubs SDK Checkpoint Store."
+  ],
+  "homepage": "https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs/azure-messaging-eventhubs",
```

Comment:
Check if other SDKs point to the umbrella service deirectory, or to a specific library within

Reply:
Hmm. Good point.